client/index.html

```diff
@@ -6,11 +6,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <script src="bundle.js" defer></script>
+    <title>Recipes API</title>
 </head>
 
 <body>
+    <section id="container">
+        <div id="logo">
+            <img src="images/google_logo.jpg" alt="google logo">
+        </div>
 
+        <form>
+            <input type="text" id="searchBar">
+            <div class="buttons">
+                <button type="button" id="resultsBtn">Google Search</button>
```

Comment:
I think making this an input rather than a button and making it type "submit" will mean that you can access the text in the search bar once you have submitted it?